lib/ansible/modules/files/find.py

```diff
@@ -355,6 +370,16 @@ def statinfo(st):
         'isgid': bool(st.st_mode & stat.S_ISGID),
     }
 
+def filterbyownerandgroup(path, uid, gid):
+    path_stat = os.stat(path)
+    if uid is not None and path_stat.st_uid == uid:
+        return True
+    elif gid is not None and path_stat.st_gid == gid:
+        return True
+    elif uid is None and gid is None:
+        return True
+    else:
+        return False
```

Comment:
We can also write a single line return statement. Please see if it makes sense.
```suggestion
        return (uid is not None and path_stat.st_uid == uid) or (gid is not None and path_stat.st_gid == gid) or (uid is None and gid is None)
```

Reply:
I dont know, Wont that make code less readable?